docs/quickstart.ipynb

```diff
@@ -4,20 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>Is a user guaranteed to have wget rather than curl?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
`wget` is more common on Linux (I don't know on OSX), in minimal VM or Docker images wget is present and curl is not.

Reply:
Great